### PR TITLE
address notes & water media

### PR DESCRIPTION
**Pull request by cristinamullin**

### CODE REVIEW

File: R/mod_query_data.R [hunk 1]
```diff
@@ -52,7 +52,7 @@ mod_query_data_ui <- function(id){
               style = "color:#0072B2;",
               title = "At present, TADA is only designed to work with water sample media"
             )
-          ), choices = c("",media), selected = "Water", multiple = TRUE)),
+          ), choices = c("",media), selected = c("Water", "water"), multiple = TRUE)),
```
Comment (cristinamullin) [inline R/mod_query_data.R:55]: I tried to add "water" to the list of autopopulated media values along with "Water". What else needs to be edited to add "water" here?

Comment (ehinman) [reply R/mod_query_data.R:55]: @cristinamullin It needed to be added to the "media" object in addition to the selectizeInput. Another approach would be to automatically look for "W"ater and "w"ater using the dataRetrieval query if someone picks the media name Water. Which would you prefer?

Comment (cristinamullin) [reply R/mod_query_data.R:55]: automatically would be nice, but then users of the WQP interface (or if they go back and forth) will not know. So I am torn... but for that reason I think we should keep both until this issue is fixed on the WQP side

Comment (ehinman) [reply R/mod_query_data.R:55]: We can keep it separated for now. It's less coding that way, haha, and more transparent. I also added in a few other steward/nwis-specific media types.

**Pull request merged**